packages/babel-plugin-transform-for-of/src/index.ts

```diff
@@ -9,6 +9,24 @@ export interface Options {
   loose?: boolean;
 }
 
+function buildLoopBody(path, declar, newBody?) {
```

Comment:
Can you add type annotations to the params here? This can save me some time :P, as I am working on enabling `noImplicitAny` compiler option.

Reply:
of course.